lib/shelley/test/data/cardano-node-shelley/shelley-genesis.yaml

```diff
@@ -16,9 +16,9 @@ protocolParams:
     major: 0
   minUTxOValue: 1000000
   decentralisationParam: 0.25 # means 75% decentralised
-  maxTxSize: 4096
+  maxTxSize: 40960
```

Comment:
Pretty sure we need raise the maxTxSize, in order to hit the TxOutputTooLarge error.

Reply:
> Pretty sure we need raise the maxTxSize, in order to hit the TxOutputTooLarge error, which also occurs at 4096.

It's my understanding that `maxTxSize` is currently `16384` bytes: https://github.com/input-output-hk/cardano-node/blob/99e678c008405426f4af774f16f4f132539622f5/configuration/cardano/mainnet-shelley-genesis.json#L13

![maxTxSize](https://user-images.githubusercontent.com/206319/110418925-49d3bc80-80d3-11eb-8359-8459e7d066b3.png)

Reply:
👍 let's use that then